src/components/landingPage/index.js

```diff
@@ -0,0 +1,270 @@
+            <div className="col-md-6 landingPage__bigArticleAreaLeft">
+              <div className="bigArticleAreaLeft-card">{displayFirstAndSecond}</div>
+            </div>
+            <div className="col-md-6 landingPage__bigArticleAreaRight">
+              <div className="bigArticleAreaRight-card">
+                <div className="popularArea-articleCard-div">{displayThirdAndFourth}</div>
+              </div>
+            </div>
+          </div>
+          <div className="row landingPage--popularArea">
+            <div className="col-md-6 landingPage__popularAreaLeft">
+              <div className="popularAreaLeft-card">
+                <div className="popularAreaLeft__upperLine">
+                  <InterestArticlesAreaUpperGreenLine />
+                </div>
+                <div className="popularArea__articlesArea">
+                  <div className="popularArea-articleCard-div">{otherArticles}</div>
+                </div>
+                <div className="landingPage__moreArticles">
+                  <LandingPageViewMoreArticles
+                    key="1"
+                    onClick={() => this.viewMoreArticles('limitArticlesNumber', remainingArticles.length, 4)}
+                    buttonMessage={
+                      limitArticlesNumber < (remainingArticles || []).length
+                        ? 'view more articles'
+                        : 'view less articles'
+                    }
+                  />
+                </div>
+              </div>
+            </div>
+            <div className="col-md-6 landingPage__popularAreaRight">
+              <div className="popularAreaRight-card">
+                <div className="popularAreaRight__upperLine">
+                  <PopularArticleAreaUpperLine />
+                </div>
+                <div className="popularArea__articlesArea">
+                  <div className="popularArea-articleCard-div">{popularArticlesRender}</div>
+                </div>
+                <div className="landingPage__moreArticles">
+                  <LandingPageViewMoreArticles
+                    key="2"
+                    onClick={() => this.viewMoreArticles('limitPopularArticlesNumber', popularArticles.length, 6)}
+                    buttonMessage={
+                      limitPopularArticlesNumber < (popularArticles || []).length
+                        ? 'view more popular articles'
+                        : 'view less articles'
+                    }
+                  />
+                </div>
+              </div>
+              <div className="popularAreaRight-freeSpace" />
+              <div className="popularAreaRight-links">
+                <LandingPageLinks />
+              </div>
+            </div>
+          </div>
+          <div className="row image-div landingPage--footer">
+            <div className="landingPage__footerArea">
+              <Footer />
+            </div>
+          </div>
+        </div>
+      </div>
+    );
+  }
+}
```

Comment:
I think codeclimate is already complaining about these many lines for a function. Is there a way we can refactor this render function to make it less complex?